string state
int64  reward
int64  done

